keyboards/handwired/sauceball/keymaps/xia0/keymap.c

```diff
@@ -0,0 +1,127 @@
+// Copyright 2023 xia0 (@xia0)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "print.h"
+#include "mousekey.h"
```

Comment:
```suggestion
```

Reply:
@fauxpark on line 57 and onwards the functions mousekey_clear, mousekey_on and mousekey_off are used. Would these be affected if mousekey.h is not included?

Reply:
It should be implicitly included via `QMK_KEYBOARD_H` (default_keyboard.h) <- quantum.h
If not, that's a bug.

Reply:
@fauxpark didn't build after removing the include with the following error:

> Compiling: quantum/keymap_introspection.c                                                          In file included from quantum/keymap_introspection.c:5:
> ./keyboards/handwired/sauceball/keymaps/xia0/keymap.c: In function 'process_record_user':
> ./keyboards/handwired/sauceball/keymaps/xia0/keymap.c:54:11: error: implicit declaration of function 'mousekey_clear'; did you mean 'timer_clear'? [-Werror=implicit-function-declaration]
>            mousekey_clear();
>            ^~~~~~~~~~~~~~
>            timer_clear
> ./keyboards/handwired/sauceball/keymaps/xia0/keymap.c:55:11: error: implicit declaration of function 'mousekey_on' [-Werror=implicit-function-declaration]
>            mousekey_on(DRAG_SCROLL_KEY);
>            ^~~~~~~~~~~
> ./keyboards/handwired/sauceball/keymaps/xia0/keymap.c:56:11: error: implicit declaration of function 'mousekey_send'; did you mean 'host_system_send'? [-Werror=implicit-function-declaration]
>            mousekey_send();
>            ^~~~~~~~~~~~~
>            host_system_send
> ./keyboards/handwired/sauceball/keymaps/xia0/keymap.c:58:11: error: implicit declaration of function 'mousekey_off' [-Werror=implicit-function-declaration]
>            mousekey_off(DRAG_SCROLL_KEY);
>            ^~~~~~~~~~~~
> cc1: all warnings being treated as errors
>  [ERRORS]
>  |
>  |
>  |
> gmake[1]: *** [builddefs/common_rules.mk:357: .build/obj_handwired_sauceball_xia0/quantum/keymap_introspection.o] Error 1
> gmake[1]: *** Waiting for unfinished jobs....

Just gonna revert the changes for now.

Reply:
Yeah, from a quick sweep of the repo it seems `mousekey.h` is not included anywhere it would get picked up through quantum.h. I'll try to fix that up in a bit.

Reply:
Oops, I kinda forgot about this. I've opened https://github.com/qmk/qmk_firmware/pull/21897 to fix it.